FutureNorthants/northants-website
# Add new icons

Add new icons to lists for Parks and Roads and Highways. 

This PR depends on the following PR's being merged, then the frontend updated: 

- https://github.com/FutureNorthants/northants-design-system/pull/121
- https://github.com/FutureNorthants/northants-design-system/pull/124

## To Test
- `./drush.wrapper @docker config:import`
- `./drush.wrapper @docker cache:rebuild`
- Edit a service page and a service landing page, the new icons should now be selectable in the Icon dropdown list.
